lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -119,6 +131,13 @@ def get_course_hierarchy(self):
                     {'#type': 'vertical', '#ref': 'G'},
                 ],
             },
+            {
+                '#type': 'sequential',
+                '#ref': 'NotASpecialExam',
```

Comment:
Why not just call this section by a Letter name?  Isn't the subsection marked `A` also Not A Special Exam?